### DISCUSSION TITLE
thank you!  but why are you warning about a 3 meg file?  :)

### DISCUSSION THREAD

**Discussion by Stephen-Hazel** [notepadqq/notepadqq#1064] [Ideas]
i mean bump that up a bit !  
(or toss the warning - finding out the hard way about stuff like that is no big deal!)

i'm editing log files all the time and it's not like i'm on a raspberry pi...

but thank you!!  I'm on kubuntu and i hate kate!  
it can't seem to remember it's window positon.
Probably my fault in some way but it's not like i haven't tried.
and the wine notepad++ is junk.

**Comment by Karnor** [comment]
Still getting warned about trivial file sizes on a system with 32GB ram, it's silly af.

I don't understand your problems with Kate though, it works fine for me wrt remembering it's screen position. It's quite a good text editor.

However it does not work on wayland, or better to say barely works on wayland which is why I'm looking for an alternative.
